docs/installation-and-operation/upgrading-metabase.md

```diff
@@ -118,3 +118,23 @@ If you're on a [Metabase Cloud](https://www.metabase.com/pricing) plan, your Met
 
 - [Upgrading AWS Elastic Beanstalk deployments](running-metabase-on-elastic-beanstalk.md#deploying-new-versions-of-metabase-on-elastic-beanstalk)
 - [Upgrading Azure Web Apps deployments](running-metabase-on-azure.md#additional-configurations)
+
+## Rolling back an upgrade
+
+In general, regular backups (especially backups before upgrading), are the best policy, so we recommend reverting to a backup of your application database to roll back an upgrade. 
```

Comment:
I think we could add more context in this paragraph. Not sure if people know why they need to rollback their database before downgrading.

I'd add something like `When Metabase upgrades to a new version, we run migrations that change the application database schema. In general, we recommend restore a backup (generated before upgrading) to roll back an upgrade`